app/src/main/res/anim/slide_in_up.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Let's do this instead:
1. Don't use an Android resource, which we don't have control over
2. Separate the /> into its own line to prevent merge conflicts
3. Move namespace to a separate line for cleanliness
4. Rename this to slide_in_from_bottom, since that's more descriptive
```
<?xml version="1.0" encoding="utf-8"?>
<translate
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:fromYDelta="100%p"
    android:toYDelta="0%p"
    android:duration="@integer/default_anim_length"
    />
```